model/src/main/java/org/mskcc/cmo/metadb/model/Library.java

```diff
@@ -204,4 +206,49 @@ public Integer getNumFastQs() {
     public String toString() {
         return ToStringBuilder.reflectionToString(this);
     }
+    
+    /**
+     * 
+     * @param libraryList
+     * @return true is Object Library is found in the list of Libraries
+     */
+    public boolean existsIn(List<Library> libraryList) {
```

Comment:
move towards introducing an overrides `equals(..)` method instead for these classes

remove list comparisons being done here